### ISSUE TITLE
[Feature Requet] Support Windows Secure Desktop

### ISSUE THREAD

**Issue by victor-rds** [winauth/winauth#286]
Hello,

I like to request, if possible, support for the windows secure desktop on password input like Keepass, this can enhance the security against software key loggers.

**Comment by My1** [comment]
yeah that would be great, especially on Laptop where hw keyloggers are hard to impossible due to lack of space.

**Comment by beerisgood** [comment]
That would be awesome!
Like it very much from KeePass.

**Comment by p1r473** [comment]
Would love this feature!

**Comment by p1r473** [comment]
Another great KeePass features we can implement- Two-Channel Auto-Type Obfuscation

http://keepass.info/help/v2/autotype_obfuscation.html

**Comment by beerisgood** [comment]
*push*

**Comment by beerisgood** [comment]
*push*

Please add this and please add https://github.com/winauth/winauth/issues/286#issuecomment-249405815
